tests/__init__.py

```diff
@@ -1043,6 +1043,13 @@ def pay(self):
     "type": "customer.created",
 }
 
+FAKE_EVENT_CUSTOMER_DELETED = (
```

Comment:
one line on this, but the update is fine on multipe. Just add a comma after the last item in the vertical list